src/pages/Home/Home.tsx

```diff
@@ -112,6 +133,13 @@ const Home: FC = () => {
                 index,
               ) => (
                 <AccommodationCard
+                  onDelete={
+                    isAdmin
+                      ? () => {
+                          if (id != undefined) onDeleteAccommodation(id);
```

Comment:
`id !== undefined`, Id in some weird scenario could be called 'undefined' then this check wouldn't pass